claim/README.md

```diff
@@ -102,45 +122,104 @@ it will use these values to attempt to connect the container, automatically addi
 Rooms. If a proxy is specified, it will be used for the connection process and for connecting to Netdata Cloud.
 
 These variables can be specified using any mechanism supported by your container tooling for setting environment
-variables inside containers. For example, when creating a new Netdata continer using `docker run`, the following
-modified version of the command can be used to set the variables:
+variables inside containers. For example, when creating a new Netdata container using `docker run`, the following
 
-```bash
+When using the `docker run` command, if you have an agent container already running, it is important to know that there will be a short period of downtime. This is due to the process of recreating the new agent container.
+
+The command that Netdata Cloud will provide to you is:
+
+```bash 
 docker run -d --name=netdata \
   -p 19999:19999 \
+  -v netdataconfig:/etc/netdata \
   -v netdatalib:/var/lib/netdata \
   -v netdatacache:/var/cache/netdata \
   -v /etc/passwd:/host/etc/passwd:ro \
   -v /etc/group:/host/etc/group:ro \
   -v /proc:/host/proc:ro \
   -v /sys:/host/sys:ro \
   -v /etc/os-release:/host/etc/os-release:ro \
-  -e NETDATA_CLAIM_TOKEN=TOKEN \
-  -e NETDATA_CLAIM_URL="https://app.netdata.cloud" \
-  -e NETDATA_CLAIM_ROOMS=ROOM1,ROOM2 \
   --restart unless-stopped \
   --cap-add SYS_PTRACE \
   --security-opt apparmor=unconfined \
-  netdata/netdata
+  -e NETDATA_CLAIM_TOKEN=TOKEN \
+  -e NETDATA_CLAIM_URL="https://app.netdata.cloud" \
+  -e NETDATA_CLAIM_ROOMS=ROOM1,ROOM2 \
+  -e NETDATA_CLAIM_PROXY=PROXY \
+ netdata/netdata
 ```
 
-Output that would be seen from the claiming script when using other methods will be present in the container logs.
+The output that would be seen from the connection process when using other methods will be present in the container logs.
 
 Using the environment variables like this to handle the connection process is the preferred method of connecting Docker containers
 as it works in the widest variety of situations and simplifies configuration management.
 
+#### Using Docker compose
+
+If you use `docker compose` you can copy the config provided by Netdata Cloud, which should be same as the one below:
+
+```bash
+version: '3'
+services:
+  netdata:
+    image: netdata/netdata
+    container_name: netdata
+  hostname: example.com # set to fqdn of host
+  ports:
+    - 19999:19999
+  restart: unless-stopped
+  cap_add:
+    - SYS_PTRACE
+  security_opt:
+    - apparmor:unconfined
+  volumes:
+    - netdataconfig:/etc/netdata
+    - netdatalib:/var/lib/netdata
+    - netdatacache:/var/cache/netdata
+    - /etc/passwd:/host/etc/passwd:ro
+    - /etc/group:/host/etc/group:ro
+    - /proc:/host/proc:ro
+    - /sys:/host/sys:ro
+    - /etc/os-release:/host/etc/os-release:ro
+  environment:
+    - NETDATA_CLAIM_TOKEN=TOKEN
+    - NETDATA_CLAIM_URL="https://app.netdata.cloud"
+    - NETDATA_CLAIM_ROOMS=ROOM1,ROOM2
```

Comment:
@hugovalente-pm not sure if we should have some draft values for the rooms and token

Reply:
@manos-saratsis if you want to place some draft values we could, I just kept the original _values_. I personally I don't see the added value of putting some dummy values there and think that could lead to confusion, since values like ROOM1 and ROOM2 and TOKEN user (should) immediately see that these are to be replaced, if we put a dummy value it could be not so clear